paparazzi/paparazzi/src/main/java/app/cash/paparazzi/Paparazzi.kt

```diff
@@ -633,10 +633,10 @@ class Paparazzi @JvmOverloads constructor(
       "androidx.lifecycle.ViewTreeLifecycleOwner"
     )
     private val hasSavedStateRegistryOwnerRuntime = isPresentInClasspath(
-      "androidx.savedstate.SavedStateRegistryOwner"
+      "androidx.savedstate.ViewTreeSavedStateRegistryOwner"
```

Comment:
`View#setViewTreeSavedStateRegistryOwner(owner)` is contained in `ViewTreeSavedStateRegistryOwner` and was introduced in [androidx.savedstate.*:1.2.0](https://developer.android.com/jetpack/androidx/releases/savedstate#1.2.0). 

Should we instead support a slightly older version of `1.1.0` by using the old `ViewTreeSavedStateRegistryOwner.set(view, owner)` API?

Reply:
in general, we lean towards only supporting later versions of dependencies.  would it be too aggressive to expect paparazzi users to be on 1.2.0+?

Reply:
I… don't have a good answer. Will requiring 1.2.0+ and calling it out in release notes be sufficient?

FWIW `1.2.0` was released in June, 2022 whereas `1.1.0` was released in Feb, 2022.

Reply:
> Will requiring 1.2.0+ and calling it out in release notes be sufficient?

I think so.  :shipit:

Reply:
🚀

Reply:
Seems fine to me, this version is already bumping the minimum kotlin version and requiring AGP 7.4.0. Can't wait forever for consumers to catch up and definitely don't want to hold anyone back from using savedstate 1.2.0 either.